### PR TITLE
Making pear/waterbubble deffable

### PR DESCRIPTION
**Pull request by jgh713**
Not ready to go in yet. Can't quite figure out why the defence isn't properly reflecting to serverside when taken off ignore/enabled in 'vshow server'. Probably just missed a variable somewhere.

### CODE REVIEW

**Comment by jgh713** [comment]
Marking #214

File: raw-svo.dict.lua [hunk 1]
```diff
@@ -1138,6 +1136,7 @@ dict = {
       end,
 
       oncompleted = function ()
+		defences.got("waterbubble")
```
Comment (keneanung) [inline raw-svo.dict.lua]: Wrong indentation.

File: raw-svo.setup.lua [hunk 2]
```diff
@@ -430,6 +430,14 @@ end)
...
   local oldnum, oldarea
   signals.gmcproominfo:connect(function (...)
+	if me then
+		if table.contains(gmcp.Room.Info.details, "underwater") then
+			me.is_underwater = true
+		else
+			me.is_underwater = false
+		end
+	end
+
```
Comment (keneanung) [inline raw-svo.setup.lua]: More indentation issues.

File: raw-svo.defs.lua [hunk 3]
```diff
@@ -93,8 +93,8 @@ defs_data = pl.OrderedMap {}
     def = "Your skin is toughened." })
   defs_data:set("chivalry defended", { nodef = true,
     def = "You are being defended by a stalwart ally." })
-  defs_data:set("pear", { nodef = true,
-    def = "You are surrounded by a pocket of air."})
+  defs_data:set("pear", { type = "general",
+	def = "You are surrounded by a pocket of air."})
```
Comment (keneanung) [inline raw-svo.defs.lua]: Also indentation issues

**Comment by jgh713** [comment]
Working fine now. 'pear' was display-only, so attempting to vkeep pear didn't know what to do with it. Changed pear references to waterbubble.

**Pull request merged**